roc_test.go

```diff
@@ -0,0 +1,53 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package stat
+
+import (
+	"math/rand"
+	"testing"
+)
+
+func TestROC(t *testing.T) {
```

Comment:
Uhhh... actually I just made them up on the fly (in my head). 
I should probably check them somewhere else I suppose?